### ISSUE TITLE
No consistency in margin left for green tick besides amount

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#19953]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open the app
2. Open any report
3. Send money to report
4. Click on send money message and using console window, observe margin-left to green tick icon besides amount on top and in message below
5. Again click on message displayed in that report to sent report
6. Observe margin left to tick icon in that report

## Expected Result:
App should maintain consistent margin between amount and green tick throughout all the reports in the flow

## Actual Result:
App maintains different margin for all the green tick instances throughout all the reports in the flow

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.22.0

**Reproducible in staging?:** Yes

**Reproducible in production?:** n/a

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/6bcb7d53-e8f2-440c-a680-4510ad425f68

https://github.com/Expensify/App/assets/93399543/00fd7b75-e068-4c55-b11a-d42c5c8feb4b

**Expensify/Expensify Issue URL:**

**Issue reported by:** @dhanashree-sawant

**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1685201350349499

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by sonialiap** [comment]
Reproducible

**Comment by kushu7** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
No consistency in margin left for green tick besides amount

### What is the root cause of that problem?
As stated in Issue. both are using different margin
### What changes do you think we should make in order to solve the problem?

we should remove `styles.moneyRequestHeaderCheckmark` and use `styles.iouPreviewBoxCheckmark` as its introduced 2 years back. it will help in maintaining code for same thing. 
both have same proprties except difference of `1px`.
We should update `MoneyRequestHeader.js` and `TaskHeader.js` to use `styles.iouPreviewBoxCheckmark` instead of `styles.moneyRequestHeaderCheckmark`.
### What alternative solutions did you explore? (Optional)
None

**Comment by Pujan92** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Inconsistency between checkmark icon margin with the label

### What is the root cause of that problem?
For `IOUPreview` we have left margin of 4px in the message whereas for the `MoneyRequestHeader` the margin of 8px(`styles.ml2`) set within [this latest PR](https://github.com/Expensify/App/pull/19524/files#diff-fc96b4ddf19074304b01f0880a2ebd169dc83f027f71d7fef93c071d5bdb6363R173-R179).

https://github.com/Expensify/App/blob/71017e42617bedefa04628bef5eb18d1b28df1fb/src/components/ReportActionItem/IOUPreview.js#L203

https://github.com/Expensify/App/blob/71017e42617bedefa04628bef5eb18d1b28df1fb/src/styles/styles.js#L2528-L2529

### What changes do you think we should make in order to solve the problem?
We can update the style to `styles.ml1` in the `MoneyRequestHeader` `MenuItem`.

https://github.com/Expensify/App/blob/71017e42617bedefa04628bef5eb18d1b28df1fb/src/components/MenuItem.js#L179-L180

OR update the margin for the class `iouPreviewBoxCheckmark` to `8px`.

**Comment by mollfpr** [comment]
We need to clarify the expected result before selecting the proposal.

|Money Request Report|Money Request Detail Header|Money Request Action|
|---|---|---|
|![money request report](https://github.com/Expensify/App/assets/25520267/9446a449-c0b6-4711-b48f-ad4d97148f19)|![money request header](https://github.com/Expensify/App/assets/25520267/a8d78c25-eb85-4fc8-92c3-fb324090cdc4)|![money request report action](https://github.com/Expensify/App/assets/25520267/382c9c90-3696-4789-9aec-3c2c88ac768a)|

@shawnborton Would you help to determine what margin should be set between the amount and checkmark icon from the above places? Thanks!

**Comment by shawnborton** [comment]
Just to clarify with @trjExpensify and @JmillsExpensify - this screenshot here shouldn't exist right? I thought we fixed this elsewhere:
![image](https://github.com/Expensify/App/assets/2319350/7e6f4586-c549-490d-a35a-6d92f6a798f4)


Then for the other screens, I think 8px makes sense but we're going to end up redoing the middle screenshot anyways, so I'm not really sure what we should do here.

**Comment by trjExpensify** [comment]
Whatcha' mean by shouldn't exist? Right now the ReportPreview in the workspace chat looks like that. We haven't made it a "card" style with all those changes just yet, scan receipt is where those are.

**Comment by shawnborton** [comment]
The report action mockup I shared shouldn't have a green checkmark. The report preview card, yes. But the report action line, no.

**Comment by trjExpensify** [comment]
Yeah, the report action line in the iou/expenseReport view shouldn't have a checkmark. The report preview (which right now is just in a message style) in the chat did, but given that we're killing it, may as well get rid of it now if you want. 👍

**Comment by mollfpr** [comment]
@shawnborton @trjExpensify We can change the margin for the first screenshot and remove the checkmark from the report action. Do you think this sounds good to you?

**Comment by shawnborton** [comment]
That works for me.

**Comment by grgia** [comment]
@mollfpr do any of these proposals work with the above suggestion? Thank you!

**Comment by mollfpr** [comment]
@grgia Not specifically, but closest and first is @Ollyws [proposal](https://github.com/Expensify/App/issues/19953#issuecomment-1572130217). We can set the value to 8px and removed the checkmark from the report action.

**Comment by trjExpensify** [comment]
>That works for me.

Same!

**Comment by mollfpr** [comment]
@grgia let's go with @Ollyws [proposal](https://github.com/Expensify/App/issues/19953#issuecomment-1572130217) which it does what requested before. 

🎀 👀 🎀 C+ reviewed!

**Comment by grgia** [comment]
Thanks for your patience, good to go @Ollyws!

**Comment by sonialiap** [comment]
Thanks for the correction, Olly! Melvin's bad math issue was brought up internally yesterday, I'm not sure if we've created an issue to fix this yet but we're aware and I expect we will fix it 👁️ (Seems to do something with how the API is reading the Z in the timezone)

**Comment by mollfpr** [comment]
> [@mollfpr] The PR that introduced the bug has been identified. Link to the PR:
 [@mollfpr] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:

No offending PR, another improvement on the IOU/money request design.

> [@mollfpr] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:

Since this is improvement on the visual, the regression step should be enough.

> [@mollfpr] Determine if we should create a regression test for this bug.
[@mollfpr] If we decide to create a [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) for the bug, please propose the regression test steps to ensure the same bug will not reach production again.

Regression step:
1. Open the app
2. Open any report
3. Send money to report
4. Click on send money message
5. **Verify** both the left margin of the checkmark in the header and in the preview box message are a consistent 8px.
6. **Verify** that the confirmation text (e.g paid $123.00 ...) does not have a checkmark.
7. Navigate to IOU Chat.
8. **Verify** that the confirmation text (e.g paid $123.00 ...) does not have a checkmark.
9. 👍 or 👎

**Comment by dhanashree-sawant** [comment]
@sonialiap Accepted, Thank you!
